src/quo/components/wallet/token_input/style.cljs

```diff
@@ -12,30 +12,22 @@
   {:padding-horizontal 20
```

Comment:
I am curious what are the styling changes for as I don't see ui-related problem in the issue description

Reply:
Well, the main issue doesn't specify UI bugs, but after fixing the main issue, some other issues started to be more noticeable. For instance:

- Visual overflow of the suffix text of the input when swapping mode to fiat and the number is too big. Needed to refactor a bit the views and styles of Token Input component.
- Don't allow the user to use more decimals than the token supports.
- Fiat character symbol was not showing in the conversion.
- Fiat character symbol was not used when showing the max amount.
- Many trailing zeroes issues when leading with large numbers.
- Routes being fetched when swapping to fiat, and vise versa

Added this in the description, will add some comments also to specify those changes.